resources/public/api/conf/1.0/reliefOther_retrieve.raml

```diff
@@ -0,0 +1,27 @@
+is:
+  - headers.acceptJson
+  - headers.contentTypeJson
+  - headers.testScenarioHeader
+  - errors.formatNino
+  - errors.formatTaxYear
+  - errors.ruleTaxYearRangeInvalid
+  - errors.clientOrAgentNotAuthorised
+  - errors.notFound
+displayName: Retrieve other reliefs [test only]
```

Comment:
Just to keep it consistent with the other endpoints could you just make "other reliefs" also start with caps.